### PR TITLE
libvirt proposal

### PR DESCRIPTION
**Pull request by xdel**
Please test everywhere.

### CODE REVIEW

File: deployment/puppet/nova/manifests/compute/libvirt.pp [hunk 1]
```diff
@@ -40,6 +40,14 @@
 
     Service['avahi-daemon'] -> Service['libvirt']
 
+    Exec['libvirt-guests-off'] -> Service <| title == 'libvirt' |>
+
+    exec {'libvirt-guests-off':
```
Comment (xenolog) [inline deployment/puppet/nova/manifests/compute/libvirt.pp]: -1

why we do not use the Service resource?

File: deployment/puppet/nova/manifests/compute/libvirt.pp [hunk 2]
```diff
@@ -49,6 +57,13 @@
       ensure => present,
       before => Package[$::nova::params::compute_package_name],
     }
+    service { 'libvirt-guests':
```
Comment (aglarendil) [inline deployment/puppet/nova/manifests/compute/libvirt.pp]: Ubuntu part is not needed. Remove it

File: deployment/puppet/nova/manifests/compute/libvirt.pp [hunk 3]
```diff
@@ -40,6 +40,14 @@
 
     Service['avahi-daemon'] -> Service['libvirt']
 
+    service { 'libvirt-guests':
+      name       => 'libvirt-guests',
+      enable     => false,
+      ensure     => true,
```
Comment (aglarendil) [inline deployment/puppet/nova/manifests/compute/libvirt.pp:46]: why this "ensure" is set to true ??

Comment (bogdando) [inline deployment/puppet/nova/manifests/compute/libvirt.pp:46]: 'stopped' maybe?

Comment (xdel) [inline deployment/puppet/nova/manifests/compute/libvirt.pp:46]: I had explained this before - script is completely stateless, so removing symlinks from every runlevel will do the same.

**Pull request merged**